builder_data.yaml

```diff
@@ -50,10 +50,10 @@ apt_repo_command: |
   keychain -k mine;
 yum_repo_command: |
   for repodir in $(find "__REPO_PATH__" -mindepth 2 -name "*.rpm" | xargs -I {} dirname {} | uniq) ; do
-    [ -d "${repodir}" ] || continue;
-    echo "Generating repodata for ${repodir}..."
-    sudo createrepo --checksum=sha --database --update "${repodir}";
-    echo "Finished generating repodata for ${repodir}..."
+    [ -d "${repodir}" ] || continue &&
+    echo "Generating and signing repodata for ${repodir} with __GPG_KEY__..." &&
+    sudo createrepo --checksum=sha --database --update "${repodir}" &&
+    gpg --no-tty --use-agent --armor --detach-sign -u __GPG_KEY__ "${repodir}/repodata/repomd.xml"
```

Comment:
@underscorgan how would you feel about not running createrepo as sudo, and instead always enforcing permissions here? `chown root:release ; chmod g+w`?